src/comps/ReclaimManager.tsx

```diff
@@ -453,6 +454,22 @@ const ReclaimDeposit = ({
               {useShortAddress(getWalletAddress() || "")}
             </p>
           </div>
+          <div className="flex flex-1 items-end">
+            <SubText>
+              Please note that we have received reports of errors with Ledgers
+              running the reclaim function - if you are using a ledger and
+              experience this please contact our support team
+              <a
```

Comment:
Nit: not sure if just in the link, but it looks like the screen includes spaces. Can you check/fix it? 
<img width="92" alt="Screenshot 2024-12-20 at 12 09 25 PM" src="https://github.com/user-attachments/assets/2ba1b8a6-6c99-415e-b279-0eb05d2b1ae0" />